packages/lesswrong/components/common/MetaInfo.jsx

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames'
 
 const styles = (theme) => ({
   root: {
-    display: "inline",
+    display: "inline-block",
```

Comment:
This is probably the riskiest thing in the commit, since it affects a large number of things that aren't part of the sidebar.